src/style/SidebarContent.scss

```diff
@@ -75,9 +75,24 @@
   }
 
   &-category-subitem {
-    margin-bottom: 3%;
+    margin: 5% 0;
+
+    .subitem {
+      color: black;
+      text-decoration: none;
+    }
+
+    .subitem-active {
+      width: 4px;
+      height: 25px;
+      border-left: 3px solid #16ac60;
+      padding-left: 16px;
```

Comment:
These are very specific and hardcoded values. Please make sure they work well across device sizes.

Reply:
Awesome